.eslintrc

```diff
@@ -0,0 +1,100 @@
+{
+	"env": {
+		"browser": true,
+		"node": true,
+	},
+	"extends": "eslint:recommended",
+	"globals": {
+		"Prism": "readonly",
+	},
+	"parserOptions": {
+		"ecmaVersion": 5,
+	},
+	"root": true,
+	"rules": {
+		"comma-dangle": [
+			"warn",
+			"only-multiline",
+		],
+//		"indent": [
+//			"error",
+//			"tab",
+//			{
+//				"SwitchCase": 1,
+//				"outerIIFEBody": 0,
+//			}
+//		],
+		"linebreak-style": [
+			"error",
+			"unix",
+		],
+		"no-console": "off",
+		"no-empty": "off",
+		"no-inner-declarations": "off",
+		"no-mixed-spaces-and-tabs": "error",
+		"no-redeclare": "off",
+		"no-sparse-arrays": "off",
+		"no-unused-vars": [
+			"warn",
+			{
+				"args": "none",
+				"varsIgnorePattern": "^_",
+			}
+		],
+		"no-useless-escape": "off",
+//		"quotes": [
+//			"warn",
+//			"single",
+//			{
+//				"avoidEscape": true,
+//			}
+//		],
+		"semi": [
+			"error",
+			"always",
+		],
+	},
+	"overrides": [
+		{
+			"files": [
+				"gulpfile.js",
+			],
+			"env": {
+				"es6": true,
+			},
+			"parserOptions": {
+				"ecmaVersion": 2015,
+			},
+			"rules": {
+				"comma-dangle": [
+					"error",
+					"always-multiline",
+				],
+			},
+		},
+		{
+			"files": [
+				"code.js",
+				"download.js",
+				"examples.js",
+				"utopia.js",
```

Comment:
I'm not really sure if we should lint this file. It feels more like a library to me.

@LeaVerou Isn't this the predecessor to [Bliss](https://github.com/LeaVerou/bliss)?

Reply:
Yes, this is the predecessor to Bliss, but it’s not a 100% drop&#x2011;in replacement.

Reply:
Wait. Don't we ignore utopia anyway? (`.eslintignore`)

Reply:
Right, we&nbsp;do…